recipes/openssl/3.x.x/conanfile.py

```diff
@@ -711,13 +711,14 @@ def _module_file_rel_path(self):
                             "conan-official-{}-variables.cmake".format(self.name))
 
     def package_info(self):
-        self.cpp_info.names["cmake_find_package"] = "OpenSSL"
-        self.cpp_info.names["cmake_find_package_multi"] = "OpenSSL"
-        self.cpp_info.names["pkg_config"] = "openssl"
+        self.cpp_info.set_property("cmake_target_name", "OpenSSL")
```

Comment:
You have forgotten `cmake_file_name`, now it generates `Findopenssl.cmake` instead of `FindOpenSSL.cmake`. Therefore test package uses built-in `FindOpenSSL.cmake` from CMake installation instead of conan generated one.